crates/bins/node/src/rest_service.rs

```diff
@@ -0,0 +1,504 @@
+#[cfg(feature = "rest")]
```

Comment:
in the parent

```
#[cfg(feature = "rest")]
mod rest;
```

And then you don't have to add it to each and every symbol present in this file